layouts/shortcodes/a.html

```diff
@@ -1 +1 @@
-<a href="{{ .Get "href" }}">{{ .Inner }}</a>
\ No newline at end of file
+<a target="{{ .Get "target" }}" rel="{{ .Get "rel" }}" href="{{ .Get "href" }}"> {{ .Inner }} </a>
```

Comment:
Don't put spaces before and after `{{ .Inner }}` in some cases this may render unnecessary whitespace.